src/Sulu/Bundle/ContentBundle/Resources/public/js/model/content.js

```diff
@@ -44,8 +44,18 @@ define([
             return this.fetch.call(this, options);
         },
 
-        fullDestroy: function(webspace, language, options) {
-            options = _.defaults((options || {}), {url: this.urlRoot + (this.get('id') !== undefined ? '/' + this.get('id') : '') + '?webspace=' + webspace + '&language=' + language});
+        fullDestroy: function(webspace, language, force, options) {
+            options = _.defaults(
+                (options || {}),
+                {
+                    url: this.urlRoot
+                        + (this.get('id') !== undefined ? '/'
```

Comment:
If `this.get('id') === undefined` there is nothing to delete. Throw error or return early?